### PR TITLE
appsso: add token viewer doc to verify token

### PR DESCRIPTION
**Pull request by sarahwalther**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

- 1.5.3
- 1.4.8
- 1.4.7

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: app-sso/how-to-guides/service-operators/verify-token-settings.hbs.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+# Verify token settings
```
Comment (mamachanko) [inline app-sso/how-to-guides/service-operators/verify-token-settings.hbs.md:1]: **[thought, non-blocking]:** In the TOC there are sections "token settings", "token signatures" and then "verify token settings". It seems as if the last relates to the first, so why does it come last? Could it be merged into "token settings" for cohesion?

File: app-sso/how-to-guides/service-operators/verify-token-settings.hbs.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+  selector:
+    matchLabels:
+      name: test-application
+  template:
+    metadata:
+      labels:
+        name: test-application
+    spec:
+      containers:
+        - image: bitnami/oauth2-proxy:7.3.0
```
Comment (mamachanko) [inline app-sso/how-to-guides/service-operators/verify-token-settings.hbs.md:69]: **[thought, non-blocking]:** I wonder if we should include a note/warning that this is pulling in an unvetted, public image even though its bitnami. Also, it won't work behind an air-gap.

**Review by mamachanko — APPROVED** [review]

File: app-sso/how-to-guides/service-operators/verify-token-settings.hbs.md (hunk 1 above)
Comment (sarahwalther) [reply app-sso/how-to-guides/service-operators/verify-token-settings.hbs.md:1]: I went back and forth on this one and didn't know which I preferred and ended up just making a decision to move forward. I'm happy to move it!

File: app-sso/how-to-guides/service-operators/verify-token-settings.hbs.md (hunk 2 above)
Comment (sarahwalther) [reply app-sso/how-to-guides/service-operators/verify-token-settings.hbs.md:69]: Great call! Yes, we definitely should include a warning. And yes, I will also add the air gap note.

**Comment by YugangW** [comment]
@sarahwalther @mamachanko I assume this change also applies to 1.5.4, 1.6.1 and 1.6.2?

**Comment by mamachanko** [comment]
> @sarahwalther @mamachanko I assume this change also applies to 1.5.4, 1.6.1 and 1.6.2?

Yes! ty

**Pull request merged**